lib/tools/sdb-commands.js

```diff
@@ -0,0 +1,98 @@
+import log from '../logger.js';
+
+let methods = {};
+
+methods.getSdbWithCorrectSdbPath = async function () {
+  this.executable.path = await this.getSdkBinaryPath("sdb");
+  this.binaries.sdb = this.executable.path;
+  return this.sdb;
+};
+
+methods.isDeviceConnected = async function () {
+  let devices = await this.getConnectedDevices();
```

Comment:
Use `sdb get-state`  to determine whethere device connected (`device`) or not (`offline` or `locked` or `unknown`).

Reply:
Done